### PR TITLE
[POA-2021] Add new sponsor and program

### PR DESCRIPTION
**Pull request by csdiedrich**
[POA-2021] Add new sponsor and program

### CODE REVIEW

**Comment by mattstratton** [comment]
There's something odd happening with the ignites as well, so let me take a look

image

**Comment by mattstratton** [comment]
@csdiedrich it looks like you don't have files in the `program/` directory for the ignites/panel which is why they aren't showing up (see the screenshot in my comment above)

**Comment by csdiedrich** [comment]
The ignites will be filled by the end of the month, we are still looking for the names.

**Comment by csdiedrich** [comment]
We don't have ignites yet, so there's no content in this section, we'll close the ignites by the end of the month

File: data/events/2021-porto-alegre.yml [hunk 1]
```diff
@@ -176,3 +179,103 @@ sponsor_levels:
...
+    start_time: "17:00"
+    end_time: "18:00"
+    background_color: "#b7da5b"
+  - title: "Encerramento e Sorteios"
+    type: custom
+    date: 2021-07-10
+    start_time: "18:00"
+    end_time: "18:30"
+
+ignites:
```
Comment (mattstratton) [inline data/events/2021-porto-alegre.yml:271]: I'm confused about what these entries are for if it's not yet selected?

**Comment by mattstratton** [comment]
well, I guess when I was trying to do the resolve of the conflicts, it ended up merging it, which is...fine?

**Comment by csdiedrich** [comment]
Great, I will adjust ignites soon. Thanks.

**Pull request merged**